Packages/io.chainsafe.web3-unity.web3auth/Runtime/WalletGUI/Scripts/Web3AuthWalletGUITxManager.cs

```diff
@@ -78,15 +79,20 @@ private void OnTransactionRequested(TransactionRequested transactionRequested)
         
         PromptTransactionRequest();
     }
-
-    private void PromptTransactionRequest()
+    
+    /// <summary>
+    /// Prompts transaction request display.
+    /// </summary>
+    private async void PromptTransactionRequest()
     {
         TransactionRequested transactionRequested = _transactionQueue.Peek();
         
         incomingTxNotification.SetActive(true);
 
         if (AutoConfirmTransactions)
         {
+            await Task.Delay(1000);
```

Comment:
in Unity version 2022 and up Coroutines are async so you can use them but for this particular problem this is the cleaner solution.

Reply:
tested and updated

Reply:
> This won't work on WebGL + there's a much cleaner solution change `Web3AuthWallet.SendTransaction(TransactionRequest transaction)` to this ![image](https://private-user-images.githubusercontent.com/36323674/342753070-1f92df1c-9f77-4a17-b2b9-50cc300826e7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkzMjM4MTQsIm5iZiI6MTcxOTMyMzUxNCwicGF0aCI6Ii8zNjMyMzY3NC8zNDI3NTMwNzAtMWY5MmRmMWMtOWY3Ny00YTE3LWIyYjktNTBjYzMwMDgyNmU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI1VDEzNTE1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIwZGZmNDI0NmFjYjYxM2JjMTgzNjI1OTQwNTM2MTkzMDEwZTU5Zjk4M2I0MDFmMGIwZGIyMjdiNjAzMjcyY2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._PjHJJxbpYv6Dx-lSHiy3HwOKBbTNFAoIN1na-JRFlw) Basically moving line 156 and 158 upwards before `transactionHandler.RequestTransaction(request);`

Nice catch, I just checked the web3auth login in WebGL.
But just in case, this is working correctly in webgl now:
```
"Evm - SendTransaction" executed successfully.
Output: 0xa353f442bcaef1d8f205c39b5c1b7c4de6313aa26ebaf0b06c95db801ffff880
```
<img width="930" alt="Screenshot 2024-06-25 at 12 29 22 PM" src="https://github.com/ChainSafe/web3.unity/assets/104365141/bd86c3d7-fb3b-4a63-959d-8ce95cbd7396">